### ISSUE TITLE
How to set the default menuButton for BOT (not for chat)?

### ISSUE THREAD

**Issue by siarheipashkevich** [defstudio/telegraph#479]
Telegram API has an ability to omit the `chat_id` parameter: https://core.telegram.org/bots/api#setchatmenubutton

But `Telegraph` facade is not allowed to call this method without `chat_id`.

<img width="590" alt="image" src="https://github.com/defstudio/telegraph/assets/7753600/028a2f8d-56f5-48c8-a6e3-e46cbfd058f3">

```
[2024-01-23 16:30:17] production.ERROR: No TelegraphChat defined for this request
```

**Comment by fabio-ivona** [comment]
Thanks for pointing this out, will fix this in next release

**Comment by fabio-ivona** [comment]
implemented in #484
